src/cascadia/LocalTests_TerminalApp/SettingsTests.cpp

```diff
@@ -1366,4 +1369,46 @@ namespace TerminalAppLocalTests
             VERIFY_ARE_EQUAL(guid3, settings._globals._defaultProfile);
         }
     }
+    void SettingsTests::TestProfileIconWithEnvVar()
+    {
+        const auto expectedPath = wil::ExpandEnvironmentStringsW<std::wstring>(L"%WINDIR%\\System32\\x_80.png");
+
+        const std::string settingsJson{ R"(
+        {
+            "profiles": [
+                {
+                    "name": "profile0",
+                    "icon": "%WINDIR%\\System32\\x_80.png"
+                }
+            ]
+        })" };
+
+        VerifyParseSucceeded(settingsJson);
+        CascadiaSettings settings;
+        settings._ParseJsonString(settingsJson, false);
+        GlobalAppSettings globalSettings{};
+        auto terminalSettings = settings._profiles.at(0).CreateTerminalSettings(globalSettings.GetColorSchemes());
+        VERIFY_ARE_EQUAL(expectedPath, terminalSettings.BackgroundImage());

```

Comment:
test is called "icon", this checks background ;P

Reply:
Not any longer. That was part of the "in-progress, ignore" commit